src/sidebar/services/test/annotations-test.js

```diff
@@ -38,11 +38,10 @@ describe('annotationService', () => {
   let fakeStreamer;
   let fakeStreamFilter;
 
+  let fakeMetadata;
```

Comment:
All of the changes within this test module here are non-meaningful code organization stuff, to fix and update mocks, e.g. The tests for the new methods have not been added yet.

Reply:
Is this comment out of date now?

Reply:
Yes